LightBulb.PlatformInterop/DeviceContext.cs

```diff
@@ -48,10 +53,46 @@ public void SetGamma(double redMultiplier, double greenMultiplier, double blueMu
         SetGammaRamp(ramp);
     }
 
+    public void SetBrightness(double brightness)
+    {
+        // Brightness is a value from 0.0 to 1.0, convert to 0-100
+        var brightnessPercentage = (uint)Math.Clamp(brightness * 100, 0, 100);
+
+        foreach (var physicalMonitor in _physicalMonitors)
+        {
+            if (!NativeMethods.SetMonitorBrightness(physicalMonitor.Handle, brightnessPercentage))
+            {
+                Debug.WriteLine(
+                    $"Failed to set brightness on monitor #{physicalMonitor.Handle}. "
+                        + $"Error {Marshal.GetLastWin32Error()}."
+                );
+            }
+        }
+    }
+
     public void ResetGamma() => SetGamma(1, 1, 1);
 
     protected override void Dispose(bool disposing)
     {
+        if (_physicalMonitors.Any())
+        {
+            // This is a bit of a hack, but we need to convert the read-only list to an array
+            var physicalMonitorsArray = _physicalMonitors.ToArray();
```

Comment:
A more efficient way to convert would be


```suggestion
            var physicalMonitorsArray = _physicalMonitors as PhysicalMonitor[] ?? _physicalMonitors.ToArray();
```